tests/test_converter.py

```diff
@@ -551,36 +545,60 @@ def test_operation_transformed_into_qubit_unitary(self, recorder):
         assert np.array_equal(recorder.queue[0].parameters[0], ex.CHGate().to_matrix())
         assert recorder.queue[0].wires == Wires([0, 1])
 
-    def test_quantum_circuit_error_by_passing_wrong_parameters(self, recorder):
-        """Tests the load method for a QuantumCircuit raises a QiskitError,
-        if the wrong type of arguments were passed."""
+    def test_qiskit_gates_to_be_deprecated(self, recorder):
+        """Tests the Qiskit gates taht will be deprecated in an upcoming Qiskit version.
+        
+        This test case can be removed once the gates are finally deprecated.
+        """
+        qc = QuantumCircuit(1, 1)
 
-        theta = Parameter('θ')
-        angle = 'some_string_instead_of_an_angle'
+        single_wire = [0]
 
-        qc = QuantumCircuit(3, 1)
-        qc.rz(theta, [0])
+        with pytest.warns(DeprecationWarning) as record:
+            with recorder:
+                qc.u1(0.1, single_wire)
+                qc.u2(0.1, 0.2, single_wire)
+                qc.u3(0.1, 0.2, 0.3, single_wire)
+
+        # check that warnings were raised
+        assert len(record) == 3
+        # check that the message matches
+        deprecation_substring = "method is deprecated"
+        assert deprecation_substring in record[0].message.args[0]
+        assert deprecation_substring in record[1].message.args[0]
+        assert deprecation_substring in record[2].message.args[0]
 
         quantum_circuit = load(qc)
+        with recorder:
+            quantum_circuit()
 
-        with pytest.raises(QiskitError):
-            with recorder:
-                quantum_circuit(params={theta: angle})
+        assert recorder.queue[0].name == 'U1'
+        assert recorder.queue[0].parameters == [0.1]
+        assert recorder.queue[0].wires == Wires(single_wire)
+
+        assert recorder.queue[1].name == 'U2'
+        assert recorder.queue[1].parameters == [0.1, 0.2]
+        assert recorder.queue[1].wires == Wires(single_wire)
+
+        assert recorder.queue[2].name == 'U3'
+        assert recorder.queue[2].parameters == [0.1, 0.2, 0.3]
+        assert recorder.queue[2].wires == Wires(single_wire)
 
-    def test_quantum_circuit_error_by_calling_wrong_parameters(self, recorder):
```

Comment:
Weird diff from GitHub, this test was removed.

It doesn't make too much sense as [parameter validation has been added to Qiskit](https://github.com/Qiskit/qiskit-terra/blob/c7c110d3d11176da3166e074a63e897d0ea02ca5/qiskit/circuit/gate.py#L235) so a string input would not work. Arrays will soon be deprecated too.